docs/operator-guides/configuring-airbyte-db.md

```diff
@@ -3,7 +3,9 @@
 Airbyte uses different objects to store internal state and metadata. This data is stored and manipulated by the various Airbyte components, but you have the ability to manage the deployment of this database in the following two ways:
 
 * Using the default Postgres database that Airbyte spins-up as part of the Docker service described in the `docker-compose.yml` file: `airbyte/db`.
-* Through a dedicated custom Postgres instance \(the `airbyte/db` is in this case unused, and can therefore be removed or de-activated from the `docker-compose.yml` file\).
+* Through a dedicated custom Postgres instance \(the `airbyte/db` is in this case unused, and can therefore be removed or de-activated from the `docker-compose.yml` file\). It's not a good practice to deploy mission-critical databases on Docker or Kubernetes. 
+Using a dedicated instance will provide more reliability to your Airbyte deployment. 
+Moreover, using a Cloud-managed Postgres instance (such as AWS RDS our GCP Cloud SQL), you will benefit from automatic backup and fine-grained sizing. You can pick a pretty small instance for a starter, as the Airbyte database will only host configurations and jobs metadata.
```

Comment:
I am not sure we can say that our db instance is pretty small. Because we track job histories in the jobs database, that database can actually grow pretty large. We had OSS users before that had trouble running file-based migration because of the total database size. So it's probably better to remove this sentence?

Reply:
Ok, I'll rephrase this to suggest that they can start small but the job tables might grow according to their usage of Airbyte.